GameData/NearFutureSpacecraft/Localization/ru.cfg

```diff
@@ -0,0 +1,150 @@
+    #LOC_NFSpacecraft_rcsblock-orbital-2way-45-1_tags = cluster control dock maneuver manoeuvre react rendezvous rotate stab steer translate orion nearfuture
+    #LOC_NFSpacecraft_rcsblock-orbital-2way-45-2_title = RX-30 Двунаправленный блок двигателей РСУ
+    #LOC_NFSpacecraft_rcsblock-orbital-2way-45-2_description = Серия блоков двигателей RX от «БАЛАНС Инжиниринг» жертвует эффективностью в атмосфере ради высокой эффективности в космосе. Модель 30 - это вариант со спаренными соплами и увеличенной тягой.
+    #LOC_NFSpacecraft_rcsblock-orbital-2way-45-2_tags = cluster control dock maneuver manoeuvre react rendezvous rotate stab steer translate orion nearfuture
+    #LOC_NFSpacecraft_rcsblock-orbital-3way-1_title = RX-15T Трёхнаправленный блок двигателей РСУ
+    #LOC_NFSpacecraft_rcsblock-orbital-3way-1_description = Серия блоков двигателей RX от «БАЛАНС Инжиниринг» жертвует эффективностью в атмосфере ради высокой эффективности в космосе. После обширных исследований, инженеры установили, что наиболее эффективна конфигурация блока с тремя соплами специфической конфигурации.
+    #LOC_NFSpacecraft_rcsblock-orbital-3way-1_tags = cluster control dock maneuver manoeuvre react rendezvous rotate stab steer translate orion nearfuture
+    #LOC_NFSpacecraft_rcsblock-orbital-4way-1_title = RX-45 Улучшенный блок двигателей РСУ
+    #LOC_NFSpacecraft_rcsblock-orbital-4way-1_description = Серия блоков двигателей RX от «БАЛАНС Инжиниринг» жертвует эффективностью в атмосфере ради высокой эффективности в космосе. Модель 45 - это классический форм-фактор, обеспечивающий тягу в 4 направлениях.
+    #LOC_NFSpacecraft_rcsblock-orbital-4way-1_tags = cluster control dock maneuver manoeuvre react rendezvous rotate stab steer translate orion nearfuture
+    #LOC_NFSpacecraft_rcsblock-orbital-5way-1_title = RX-55 Улучшенный блок двигателей РСУ
+    #LOC_NFSpacecraft_rcsblock-orbital-5way-1_description = Серия блоков двигателей RX от «БАЛАНС Инжиниринг» жертвует эффективностью в атмосфере ради высокой эффективности в космосе. This block is a jack of all trades with five nozzles, allowing thrust in five directions.
+    #LOC_NFSpacecraft_rcsblock-orbital-5way-1_tags = cluster control dock maneuver manoeuvre react rendezvous rotate stab steer translate orion nearfuture
+    #LOC_NFSpacecraft_rcsblock-orbital-linear-1_title = RX-1 Линейный двигатель РСУ
+    #LOC_NFSpacecraft_rcsblock-orbital-linear-1_description = Одиночное сопло серии RX обеспечивает тягу в одном направлении. Прикрепите несколько, ну, и мир - ваш неопознанный моллюск.		// Literal translation. I can't neither understand nor adapt/localize such weird humor... if it IS a humor in the first place... =o_O=
+    #LOC_NFSpacecraft_rcsblock-orbital-linear-1_tags = control dock maneuver manoeuvre react rendezvous rotate stab steer translate orion nearfuture
```

Comment:
Hm-m-m... that Shaekspir's phrase was never translated literally. "Everything is in my hands" is most common translation. No wonder I couldn't find a clue.